Sources/Sentry/Public/SentryOptions.h

```diff
@@ -766,6 +766,29 @@ NS_SWIFT_NAME(Options)
  */
 @property (nonatomic, readonly) SentryExperimentalOptions *experimental;
 
+#if TARGET_OS_IOS && SENTRY_HAS_UIKIT
+
+/**
+ * Block used to configure the user feedback widget, form, behaviors and submission data.
+ */
+API_AVAILABLE(ios(13.0))
+typedef void (^SentryUserFeedbackConfigurationBlock)(
+    SentryUserFeedbackConfiguration *_Nonnull configuration);
```

Comment:
`l`: We have all the other typedefs for the callbacks in the options in SentryDefines.h. You could move it there for consistency.